lib/app/custom_widgets/event_text_with_icon.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:score_live/app/custom_widgets/events_widgets/card_event_widget.dart';
+import 'package:score_live/app/custom_widgets/events_widgets/goal_event_widget.dart';
+import 'package:score_live/app/custom_widgets/events_widgets/substitution_event_widget.dart';
+import 'package:score_live/app/custom_widgets/events_widgets/var_event_widget.dart';
+import 'package:score_live/models/match_events_model/match_events_model.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+import 'package:score_live/presentation/constants/app_const_variables.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
+
+class EventTextWithtIcon extends StatelessWidget {
+  const EventTextWithtIcon({
+    super.key,
+    this.event,
+    this.assist,
+    required this.time,
+    required this.player,
+    required this.isHomeTeam,
+  });
+
+  final MatchEventsModel? event;
+  final String? assist;
+  final String time;
+  final String player;
+  final bool isHomeTeam;
+
+  @override
+  Widget build(BuildContext context) {
+    final eventType = event?.type ?? AppConstVariables.stringPlaceholder;
+    final eventDetail = event?.detail ?? AppConstVariables.stringPlaceholder;
+    final playerIn = assist ?? AppConstVariables.stringPlaceholder;
+    return Row(
+      mainAxisAlignment: isHomeTeam == true ? MainAxisAlignment.start : MainAxisAlignment.end,
+      children: [
+        if (isHomeTeam == true) ...[
```

Comment:
`isHometeam` is not nullable, change to if (isHomeTeam) ... []

Reply:
you're totally right